featurebyte/query_graph/node/generic.py

```diff
@@ -1267,6 +1273,8 @@ class JoinNodeParameters(FeatureByteBaseModel):
     metadata: Optional[Union[JoinMetadata, JoinEventTableAttributesMetadata]] = Field(
         default=None
     )  # DEV-556: should be compulsory
+    left_on_timestamp_schema: Optional[TimestampSchema] = Field(default=None)
+    right_on_timestamp_schema: Optional[TimestampSchema] = Field(default=None)
```

Comment:
I think these are not necessary since the join keys will not be timestamp type. It is needed in `SCDJoinParameters` though, which is already in place 👍